vendor/brightray/browser/network_delegate.cc

```diff
@@ -144,4 +144,24 @@ bool NetworkDelegate::OnCancelURLRequestWithPolicyViolatingReferrerHeader(
   return false;
 }
 
+bool NetworkDelegate::OnCanQueueReportingReport(
+    const url::Origin& origin) const {
+  return true;
```

Comment:
I just looked at how the other functions were implemented and, similarly, as how [chrome_network_delegate.cc](https://chromium.googlesource.com/chromium/src/+/master/chrome/browser/net/chrome_network_delegate.cc) delegates to cookie_settings_->IsCookieAccessAllowed().